src/main/java/appeng/block/AEBaseBlock.java

```diff
@@ -584,4 +585,22 @@ public ItemMeshDefinition getItemMeshDefinition()
 		return null;
 	}
 
+	/**
+	 * @return Null if no special block color handler is needed, otherwise the handler to register.
+	 */
+	@SideOnly( Side.CLIENT )
+	public IBlockColor getBlockColor()
+	{
+		return null;
+	}
+
+	/**
+	 * @return Null if no special item color handler is needed, otherwise the handler to register.
+	 */
+	@SideOnly( Side.CLIENT )
+	public IItemColor getItemColor()
```

Comment:
Nah that's not true. I tested it, without the IItemColor, the item will just be white. There's no block state for which it could call the IBlockColor.